docs/source/support/roadmap.rst

```diff
@@ -1,45 +1,42 @@
 .. Copyright (c) 2019 The DAML Authors. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-DAML roadmap (as of September 2019)
+DAML roadmap (as of January 2020)
 ===================================
 
 This page specifies the major features we're planning to add next to the DAML Ecosystem. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
 
 We plan to update this roadmap roughly every three months.
 
-- **DAML Triggers**
+- **Developer Experience (DX)**
 
-  Support for non-transactional automation written directly in DAML. See `discusison on GitHub <https://github.com/digital-asset/daml/issues/1615>`_
+  - Improved version of the “Getting Started Guide” (former “Quickstart”) that will include instructions on how to build an end-to-end application with a UI
+  - An end-to-end skeleton application that can be installed via DAML Assistant
+  - Improved compatibility between different DAML SDK versions
 
-- **Developer tooling**
+- **Front-end support**
 
-  Improved developer experience and functionality on
+  - A front-end JavaScript/TypeScript framework including supporting libraries and tools that facilitate interaction with data on a DAML ledger
+  - React library built on top of the above-mentioned framework that binds ledger data to React components
 
-  - Package Management
-  - Party Allocation and Management
-  - Application Deployment
-  - Application Upgrading
+- **New “how-tos” in the docs**
 
-- **API Authentication**
+  - How to deploy DAML models to existing Ledgers
+  - How to deploy DAML full-stack DAML applications 
+  - How to migrate a DAML model to a new version
 
-  Addition of JWT based authentication to the Ledger API
+- **Ledger model improvements**
 
-- **High Level API**
+  - Improved Ledger Time model that works seamlessly in a distributed setting, and contains optional safeguards against duplicate command submission.
+  - Better privacy controls by separating the disclosure mechanisms for business and validation purposes and adding new mechanisms to share single contracts and delegate read rights 
 
-  A HTTP/JSON-based high-level API with querying capabilities
-
-- **Ledger Ops Tooling**
+- **Deployment Options**
 
-  Tooling for monitoring, logging and health checking ledgers
+  - ALPHA version of DAML on Corda
 
 - **Canton**
```

Comment:
I thought we want to stop using the name “Canton” externally?

Reply:
Didn't get any feedback on that :( 

@bame-da Any input?

Reply:
Still discussing that. Please change to "DAML Interoperability"
It's okay for now to keep Canton in the bullet points

Reply:
@nemanja-da You need to ping the Canton team explicitly. They work in a different repo and hence probably don't get notifications here. @daravep is the person to ping for things like this one.

Reply:
@shaul-da Ratko sent me a message and asked me to remove Canton updates

@hurryabit Thnx! Did got it touch with the team eventually :)